tests/helpers/setup-globals.js

```
@@ -0,0 +1,25 @@
+globalThis.document = {
+  documentElement: {
+    classList: {
+      toggle: () => {}
+    }
+  },
+  getElementById: () => null,
+  querySelectorAll: () => [],
+  addEventListener: () => {}
+};
+
+globalThis.ORGS = require('../../src/js/org.js');
```

ESLint

[error] 'require' is not defined.

(no-undef)